### DISCUSSION TITLE
Startup process / UX

### DISCUSSION THREAD

**Discussion by AlmasB** [gluonhq/scenebuilder#440] [Ideas]
SB currently opens the splash menu + the editor window. Given the editor window is not usable at this stage, perhaps we can show only the splash menu. After the user has chosen to create a new or open an existing project, then we show the editor as normal. 

From the UX perspective, it can be perceived as a cleaner approach since the user only needs to focus on one item -- the menu.

Plus, it would be good to discuss how to improve startup times and the by-product of the above change might be of benefit here.

**Comment by Oliver-Loeffler** [comment]
I was playing around with the new startup process and yes, there is a measurable and feelable improvement of startup speed.
Also directing the focus of the user to the possible actions and tasks is a valid point.

But, the welcome screen has some issues.

* Beside icon and dialog name, there is no indication that the screen belongs to Scene Builder, it looks kind of empty (so I'd welcome a larger Scene Builder icon including the name of the app in a larger label somewhere in the dialog). Scene Builder should identify itself clearly even in the welcome screen.
* Scrolling behavior is broken. If you decrease window size, somewhen all contents is clipped. Either the dialog becomes fully responsive or we somehow establish an effective minimum size for the dialog to remain usable.

There are issues where users ask that Scene Builder will remember its window position and size. Also a start-over (skipping the welcome dialog) is asked for. So if the start-over issue is worked on and accepted, this option should be accessible from the welcome screen.
